src/utils/discordUtil.ts

```diff
@@ -0,0 +1,33 @@
+import {Message, Role} from 'discord.js';
+import logger from './loggerUtil';
+import {getSettings} from '../database/models/SettingsModel';
+
+export const addRoleToAuthor = async (message: Message, roleName: string) => {
```

Comment:
Hmmm, any chance we can make this generic and pass in a user ID we want to add a role to?

Reply:
I think it's kind of limiting on a function to only be able to add a role to the author when we can pass that in as an argument and then use the same function for verification etc